operators/nvidia_video_codec/README.md

```diff
@@ -0,0 +1,17 @@
+# NVIDIA Video Codec Operators
+
+This directory includes the `nv_video_decoder` and `nv_video_encoder` operators, which are based on the 
+[NVIDIA Video Codec SDK](https://developer.nvidia.com/video-codec-sdk).
+
+These encoder and decoder operators are designed for streaming applications. The encoded frames are stored on the host (CPU) 
+memory, where they can be copied to another network streaming operator.
```

Comment:
Any feature support for GPU direct / RDMA to explore in the future with the SDK?